tools/gmhazard_scripts/db_creation/empirical_db/calc_emp_flt.py

```diff
@@ -95,6 +101,7 @@ def parse_args():
     parser.add_argument("site_source_db")
     parser.add_argument("vs30_file")
     parser.add_argument("output_dir")
+    parser.add_argument("station_file")
```

Comment:
We previously avoided having the station file as an input at this step because everything that was needed was in the site_source_db.